sdk/containerregistry/container-registry/samples/v1/javascript/deleteImages.js

```diff
@@ -31,7 +31,7 @@ async function main() {
     let imageCount = 0;
     // Delete images older than the first three.
     for await (const manifest of imageManifests) {
-      if (imageCount++ > imagesToKeep) {
+      if (imageCount++ >= imagesToKeep) {
```

Comment:
for clarity I would recommend just incrementing imageCount separately but up to you

Reply:
Good idea!